client/components/domains/domain-skip-suggestion/index.jsx

```diff
@@ -0,0 +1,53 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import DomainSuggestion from 'components/domains/domain-suggestion';
+
+class DomainSkipSuggestion extends React.Component {
+	static propTypes = {
+		onButtonClick: PropTypes.func.isRequired,
+	};
+
+	onButtonClick = () => {
+		this.props.onButtonClick( this.props.suggestion );
+	};
+
+	render() {
+		const { translate } = this.props;
+		const buttonContent = translate( 'Skip Purchase', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?